### DISCUSSION TITLE
Dreame L10 Pro not reachable trough app when docked

### DISCUSSION THREAD

**Discussion by DreameUser2000** [Hypfer/Valetudo#2123] [Q&A / Support] [answered]
### Are you already using Valetudo?

Yes, for a long time

### Your questions

Hi,

i'm using Valetudo on my Dreame L10 Pro for nearly 2 1/2 years and everything was fine until 2 to 3 month ago.

At first I thought it was a WLAN problem but since the problem didn't went away i'm asking you guys if anyone might have any idea.

The Problem:
When the robot is docked it is not/or barely reachable via the valetudo app.
It just doesn't show up in the app and it seems that it is reconnecting to the WLAN every few minutes (router log says: connected... 2 minutes later disconnected... connected ... 2 minutes later and so on).

And now the twist(s):
1.) If I wait a few minutes when the app is opened the Robot apears again.
When I select the robot the "loading bar" of the browser appears and then I get a timeout.

But if I try this a few times (go back, select the robot again, wait for timeout, go back... and so on) the overlay opens as normal and everything is fine.
I can select everything I can start the cleaning, I can view the map, set timers and so on.

2.) If the robot is cleaning I get instant access and everything is fine. I can even close the app completly and open it 20 minutes later and everything is fine.

But as soon as the robot is docked again I have the same problem after a short time.
As if the robot goes into a deepsleep.

Note:
-English is not my first language, so if you dont understamd everything or need any more information feel free to ask :)

-I only did valetudo updates every 1 to 1/2 year so it *could* be possible that a valetudo version from early/mid 2024 may be the cause (I doubt it, but I didn't want to leave it unmentioned)

-The last time I got into flashing/ssh connection and so on was the time I flashed valetudo. So if I should read logs and so on i'm shure I would need to start from the scratch.

Robot
Dreame L10 Pro
Valetudo 2024.10
Firmware 1105

WLAN/Router
FritzBox 7590
2,4 GHz b+g+n, Channel 13
3-4 devices connected to the WLAN and no Problems with the other devices
Fixed IP adress for every device
NTP via the fritz box
Internet access for the robot is blocked

I think I already tried changing every setting of the fritzbox I mentioned here but if you see someting that I should try, i'm open for suggestions :)

**Comment by Hypfer** [comment]
Did you perhaps enable WPA3 recently? Did WPA2+WPA3 enable itself recently due to an update or similar?

These bots don't seem to like it very much

**Comment by DreameUser2000** [comment]
No, it's set to
"WPA 2 (CCMP)"

I think I tried WPA 2 + WPA 3 Mode once but my telephone didn't like it either and was unable to connect at all.

**Comment by DreameUser2000** [comment, marked as answer]
Sooooo... I found the solution.

Since the problem occured I already tried diffenrent thinks, like rebooting the robot or changing wifi settings (switched channels for example....).

But I never shut down the robot for a few hours and changed the wifi channel during this time.

I am a little bit afraid to do more research, cause I am glad, that everything works normal again.

But I think the dreamie doesn't like the WiFi Channel No. 13 and it seems like it has to be turned of during the change of the WiFi channel (????).

Summary: Turn dreamie of, change wifi channel (to 3 for example), turn dreamie on again.
